ehr/resources/schemas/dbscripts/postgresql/ehr-24.000-24.001.sql

```diff
@@ -0,0 +1,17 @@
+CREATE FUNCTION ehr.handleUpgrade() RETURNS VOID AS $$
+DECLARE
+BEGIN
+      IF NOT EXISTS (
+        SELECT column_name
+        FROM information_schema.columns
+        WHERE table_name='notificationrecipients' and table_schema='ehr' and column_name='lsid'
+      )
+      THEN
+        ALTER TABLE ehr.notificationrecipients ADD COLUMN LSID LSIDtype;
+      END IF;
+END;
+$$ LANGUAGE plpgsql;
+
+SELECT ehr.handleUpgrade();
+
+DROP FUNCTION ehr.handleUpgrade();
```

Comment:
This approach should work fine, but FYI PostgreSQL has ALTER TABLE ADD COLUMN IF EXISTS. https://www.postgresql.org/docs/11/sql-altertable.html